pkg/apis/storagepool/config/cns.vmware.com_storagepools.yaml

```diff
@@ -15,93 +15,92 @@ spec:
     plural: storagepools
     singular: storagepool
   scope: Namespaced
```

Comment:
This should cluster scoped. Please replace with "Cluster"

Reply:
The CRD is just used as an example. I think it was never used for anything. However, if you see the code when CRD is created, it was cluster scoped.

Reply:
Ack